novi-sajt/css/pages/rezervacija.css

```
@@ -0,0 +1,380 @@
+
+.reservation-section {
+  background: #0f2538;
+  padding: 40px 0 60px;
+}
+
+.reservation-info-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 28px;
+
+  margin: 30px auto 60px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)